unlock-protocol-com/src/config/unlock.ts

```diff
@@ -7,21 +7,21 @@ const devConfig: Config = {
     process.env.NEXT_PUBLIC_URL_BASE ?? 'https://staging.unlock-protocol.com',
   appURL:
     process.env.NEXT_PUBLIC_UNLOCK_APP_URL ??
-    'https://staging-app.unlock-protocol.com/locks',
+    'https://staging-app.unlock-protocol.com',
```

Comment:
Redirects to front page no?

Reply:
Yeah but we don't use it directly :) This way we can customize it!